File: Angular/src/app/_models/Post_Order_Item/Proposal.ts

export class Proposal {
  constructor(
    public Id:Number,
    public Price:number,
    // public PrepTimeMin:Date,
    public Description:string,
    // public PrepTimeMin:number,
    // public Media:string,
    public PostId:Number,
    public ChefId:string,
    public datePosted:number,
    public chefName:string="",
    public chefRating:number=1,
    public chefPhoto:string="",
    ){}
}

File: Web Api/DAL/Models/Proposal.cs
using System;
using System.Collections.Generic;

namespace DAL.Models;

public partial class Proposal
{
    public int Id { get; set; }

    public int Price { get; set; }

    public string? Description { get; set; }

    public DateTime Date { get; set; } = DateTime.Now;

    public int PostId { get; set; }

    public string ChefId { get; set; } = string.Empty;

    public virtual Chef Chef { get; set; } = null!;

    public virtual Post Post { get; set; } = null!;
    public virtual PostAcceptedOrder PostAcceptedOrder { get; set; }

}
